### PR TITLE
Support mix of context and plural

### PR DESCRIPTION
**Pull request by artemtsvilii-lh**
# Description
The i18next support mix of `context` and `count` but we don't.
https://www.i18next.com/translation-function/context
Adding this option:
```
t('boardBasis.code', { context: 'AI' }) -> 'All Inclusive hotel'
t('boardBasis.code', { context: 'AI', count: 1  }) -> 'All Inclusive hotel'
t('boardBasis.code', { context: 'AI', count: 2 }) -> 'All Inclusive hotels'
```

Fixes # (issue)

### CODE REVIEW

File: src/processTranslation.spec.tsx [hunk 1]
```diff
@@ -15,6 +15,8 @@ describe('processTranslation', () => {
       [ 'reviews', { count: 123 }, '123 reviews' ],
       [ 'boardBasis.code', {}, 'Board Basis' ],
       [ 'boardBasis.code', { context: 'AI' }, 'All Inclusive' ],
+      [ 'boardBasis.code', { context: 'AI', count: 1 }, 'All Inclusive' ],
+      [ 'boardBasis.code', { context: 'AI', count: 2 }, 'All Inclusive plural' ],
```
Comment (yl-lh) [inline src/processTranslation.spec.tsx:19]: `All Inclusive plural` lol what an example

**Review by yl-lh — APPROVED** [review]

File: src/processTranslation.tsx [hunk 2]
```diff
@@ -49,14 +49,11 @@ export const processTranslation = ({
     throw new Error(`Missing translation: "${key}"`);
   }
 
-  let suffix;
-
-  if (args.context) {
-    suffix = args.context;
+  let suffix = args.context;
 
   // @TODO: Update to the new format https://www.i18next.com/translation-function/plurals
```
Comment (rahulk94) [inline src/processTranslation.tsx:54]: Is this TODO still relvant?

File: src/processTranslation.tsx [hunk 3]
```diff
@@ -49,14 +49,11 @@ export const processTranslation = ({
...
-  } else if (typeof args[COUNT] !== 'undefined' && args[COUNT] !== 1 && args[COUNT] !== -1) {
-    suffix = 'plural';
+  if (typeof args[COUNT] !== 'undefined' && args[COUNT] !== 1 && args[COUNT] !== -1) {
+    suffix = suffix ? `${suffix}_plural` : 'plural';
```
Comment (rahulk94) [inline src/processTranslation.tsx:56]: Is this breaking in that consumers need to check now that their context'd translations have a `_plural` translation present as well?

File: src/processTranslation.spec.tsx (hunk 1 above)
Comment (artemtsvilii-lh) [reply src/processTranslation.spec.tsx:19]: Changed to "All inclusive hotels"

File: src/processTranslation.tsx (hunk 2 above)
Comment (artemtsvilii-lh) [reply src/processTranslation.tsx:54]: Yes, I think:
We have 
```
single: `key`
plural: `key_plural`
```
But the i18next format is:
```
keys

{
  "key_one": "item",
  "key_other": "items",
  "keyWithCount_one": "{{count}} item",
  "keyWithCount_other": "{{count}} items"
}
sample

i18next.t('key', {count: 0}); // -> "items"
i18next.t('key', {count: 1}); // -> "item"
i18next.t('key', {count: 5}); // -> "items"
i18next.t('key', {count: 100}); // -> "items"
i18next.t('keyWithCount', {count: 0}); // -> "0 items"
i18next.t('keyWithCount', {count: 1}); // -> "1 item"
i18next.t('keyWithCount', {count: 5}); // -> "5 items"
i18next.t('keyWithCount', {count: 100}); // -> "100 items"
```

File: src/processTranslation.tsx (hunk 3 above)
Comment (artemtsvilii-lh) [reply src/processTranslation.tsx:56]: No, it's not a breaking change, the logic is the same as before. 
Line 59 has a fallback to `root[lastPart]` when there is no `_plural`

Comment (rahulk94) [reply src/processTranslation.tsx:56]: Thanks for clarifying

**Review by rahulk94 — APPROVED** [review]

**Pull request merged**